src/main/java/org/folio/converter/hld2cdx/ContributorConverter.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.converter.hld2cdx;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.lang.NonNull;
+
+import org.folio.rest.jaxrs.model.Contributor;
+
+public class ContributorConverter implements Converter<org.folio.rmapi.model.Contributor, Contributor> {
+
+  @Override
+  public Contributor convert(@NonNull org.folio.rmapi.model.Contributor source) {
+    return new Contributor()
+      .withName(source.titleContributor)
+      .withType(source.type);
+  }
+
```

Comment:
What are the benefits of using spring Converter here vs what was already in place for conversion? Prior seemed to be simple and straightforward - are we adding more dependencies with this?

Reply:
please see my [comments](https://github.com/folio-org/mod-codex-ekb/pull/86#discussion_r258817363) above.

Reply:
Ok thanks for the explanation @dmtkachenko